packages/gestalt/src/messaging.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+
+export const MESSAGING_TYPE_ATTRIBUTES = Object.freeze({
+  neutral: {
+    backgroundColor: 'secondary',
+  },
+  success: {
+    icon: 'info-circle',
+    iconColor: 'success',
+    color: 'successBase',
+    backgroundColor: 'successWeak',
+  },
+  info: {
+    icon: 'info-circle',
+    iconColor: 'info',
+    color: 'infoBase',
+    backgroundColor: 'infoWeak',
+  },
+  warning: {
+    icon: 'workflow-status-warning',
+    iconColor: 'warning',
+    color: 'warningBase',
+    backgroundColor: 'warningWeak',
+  },
+  error: {
+    icon: 'workflow-status-problem',
+    iconColor: 'error',
+    color: 'errorBase',
+    backgroundColor: 'errorWeak',
+  },
+});
+
+export const UNDEFINED = undefined;
```

Comment:
@AlbertCarreras Could we remove `export const UNDEFINED = undefined;`? Seems like a hacky way to get rid of the `import/prefer-default-export` error.

We could:
* Disable the linter: makes it explicit what's going on. The `UNDEFINED` export is confusing.
* Rename the file to `MESSAGING_TYPE_ATTRIBUTES.js` and use a default export

Reply:
DONE